app/Http/Controllers/SensorController.php

```diff
@@ -0,0 +1,99 @@
+<?php 
+
+namespace App\Http\Controllers;
+
+use App\Sensor;
+use App\DataTables\SensorDataTable;
+use Illuminate\Http\Request;
+
+class SensorController extends Controller 
+{
+    /**
+    * Create a new controller instance.
+    *
+    * @return void
+    */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+    * Display a listing of the resource.
+    *
+    * @return Response
+    */
+    public function index(SensorDataTable $dataTable)
+    {
+        return $dataTable->render('sensor.index');
+    }
+
+    /**
+    * Show the form for creating a new resource.
+    *
+    * @return Response
+    */
+    public function create()
+    {
+        return view('sensor.create');
```

Comment:
Since the create, edit, store, update, destroy functions are not being used on the both the SensorController and SensordataController. Can you just add a redirect to their index page or (even though they are not really needed) make them functional. Note: I created/updated the views and controllers for using the create, edit, store, and update so I could test this pull request without using the actual api calls. So if you go the route of making the functions functional let me know and I can commit what I have so you don't have to create all of it.

Reply:
These are what I men't about more in the coming days. I should be able to get to this tomorrow.

Reply:
all CRUD implimented with 62c2d3b